pkgs/by-name/li/linkding/common.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, fetchFromGitHub }:
+let
+  version = "v1.24.0";
+in
+{
+  inherit version;
+
+  npmDepsHash = "sha256-Ku8bCS0PHtbfzrlSnaAWMr315NYKcXe/Goiz/ZouRLk=";
+
+  src = fetchFromGitHub {
+    owner = "sissbruecker";
+    repo = "linkding";
+    rev = "38204c87cf6fdd3d871a115d9786db73b17d1be6";
```

Comment:
This is not v1.24.0. This includes [a change](https://github.com/sissbruecker/linkding/pull/620/files#diff-8ac232cb450a319a89aedfc5311c9e04edd3fea86c7db4201374a36b0e6deabdR22) that generates a random SECRET_KEY if it can't be written, previous to this we'd need another patch to avoid NixOS using the vulnerable hardcoded development key.

The differences from 1.24.0 are small, I'm not sure if we want to wait for a release or use this specific commit as a patch instead